.github/workflows/test.yml

```diff
@@ -44,3 +44,31 @@ jobs:
         with:
           coverageLocations: cover.out:gocov
           prefix: github.com/${{ github.repository }}
+
+  test-crossplane:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set cloudscale token
+        env:
+          CLOUDSCALE_API_TOKEN: ${{ secrets.CLOUDSCALE_API_TOKEN }}
```

Comment:
do we actually need a cloudscale token to test the crossplane part? We don't really check with KUTTL if the bucket is acutally there.

Reply:
that's a good question, the problem is that `local-install` does not work without the token. @ccremer does it make sense to remote the variable or set it at a later stage with makefile?

`Release "provider-cloudscale" does not exist. Installing it now.
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: unknown object type "nil" in Secret.stringData.CLOUDSCALE_API_TOKEN
make: *** [local-install] Error 1`

Reply:
Also note that secrets won't be exposed to PRs if foreign contributors make a PR. The tests would fail.
Also, without cleaning them up even after failures, I would actually consider not having these tests run as part of github actions and rather use only for local testing. Too much can fail that could cause costs from cloudscale.

Reply:
I believe we need to find a way and remove Cloudscale (the env variable) dependency rather than remove tests from github workflow altogether.

Reply:
Sadly that means not making API requests to Cloudscale from the tests altogether, which kinda defeats the purpose of the tests.
We could consider supporting some sort of Dry-run in the provider itself, where the API requests are faked, to allow making contributions from foreign sources.
But that's outside the scope here. So I'd be fine to add a comment that test run will fail if the secret is not exposed and do tests ourselves locally in the rare event where a non-vshneer makes a PR.